modules/navbar.js

```diff
@@ -0,0 +1,27 @@
+// const listNav = document.getElementById('list-page');
+// const addNewNav = document.getElementById('add-list');
+// const contactNav = document.getElementById('contact-list');
+
+const booklistSection = document.getElementById('landing-page');
+const addbookSection = document.getElementById('add-book');
+const contactSection = document.getElementById('contact-section');
```

Comment:
- Kindly, the current date has to be displayed below the navigation bar, for this requirement use the [Luxon](https://github.com/moment/luxon/blob/master/docs/install.md) library. 👍

Reply:
Hi! @binhussen  already fix the first comment but wanna check for this, I already use the luxon library you can checkout [here](https://github.com/enmanuelbayen/Awesome-Books-ES6/blob/3c917d7e76632b00aea4e9d8cab2ef56eceb16a1/index.js#L17)